src/PHPCR/NodeInterface.php

```diff
@@ -485,9 +487,8 @@ public function getNode($relPath);
      * @param string|array $typeFilter a filter or an array of filters for the
      *      node type names to find.
      *
-     * @return Iterator over all (matching) child Nodes implementing
-     *      <b>SeekableIterator</b> and <b>Countable</b>. Keys are the Node
-     *      names, values the corresponding NodeInterface instances.
+     * @return Iterator<string, NodeTypeInterface> over all (matching) child Nodes implementing <b>SeekableIterator</b>
```

Comment:
```suggestion
     * @return Iterator<string, NodeInterface> over all (matching) child Nodes implementing <b>SeekableIterator</b>
```

Reply:
autocomplete gone wrong. this should be NodeInterface indeed, you are right.

Reply:
fixed